mlflow/store/artifact/s3_artifact_repo.py

```diff
@@ -190,10 +190,6 @@ def log_artifacts(self, local_dir, artifact_path=None):
                 rel_path = relative_path_to_artifact_path(rel_path)
                 upload_path = posixpath.join(dest_path, rel_path)
 
-            if not filenames:
-                # We're in an empty directory. Create a folder to preserve the directory structure.
-                s3_client.put_object(Bucket=bucket, Key=upload_path + "/")
```

Comment:
I remember I made this change because nog logging empty folders break `sentence_transformers` tests.

Reply:
`test_load_from_remote_uri` was the failing test but it's passing now.